sass/pages/_scheduler.scss

```diff
@@ -0,0 +1,57 @@
+
+.scheduler-grid {
+    display: table;
+    table-layout: fixed;
+    width: 100%;
+    background: white;
+
+    .scheduler-row {
+        display: table-row;
+
+        .row-info, .row-grid {
+            display: table-cell;
+            border-bottom: 1px solid #f0f0f0;
+            min-height: 46px;
+        }
+
+        .row-info {
+            width: 180px;
+            vertical-align: middle;
+            padding-left: 10px;
+        }
+
+        .week-row {
+            display: table;
+            empty-cells: show;
+            table-layout: fixed;
+            width: 100%;
+            min-height: 40px;
+        }
+        .day-cell {
+            display: table-cell;
```

Comment:
While I agree with that, it's very hard to make it look decent on mobile. Bootstrap has support for responsive tables, which would be handy.